yaksh/views.py

```diff
@@ -3106,9 +3127,30 @@ def view_module(request, module_id, course_id, msg=None):
     context['course'] = course
     context['state'] = "module"
     context['msg'] = msg
+    course_status = CourseStatus.objects.filter(
+            course_id=course.id, user_id=user.id)
+    print(user.get_full_name(), course , user.email)
+    #print(course_status.first().get_certificateStatus())
+    if course.percent_completed(user, all_modules) == 100 and course_status.first().get_certificateStatus() == '':
```

Comment:
`get_certificateStatus()` should give a boolean value and not a string. If you set the certificate to the BooleanField that will be solved.